### ISSUE TITLE
Support load [default] section in credentials in "~/.aws/"

### ISSUE THREAD

**Issue by phuonghuynh** [riboseinc/aws-authenticating-secgroup-scripts#4]
We can load [default] section in  "~/.aws" so that required argument "-credentials" can be optional.

**Comment by ronaldtse** [comment]
Agree, the script should actually use AWS STS to utilize temporary tokens. Can you help update it? Thanks!

**Comment by ronaldtse** [comment]
And it should respect the AWS_PROFILE environment variable. I wonder if there’s an easy way to do so without having us to read it?

**Comment by phuonghuynh** [comment]
I agree, we could find an easy way to do this. Otherwise, we need to read AWS document to see what profile is considered in order
